app/js/HelloWorldSceneAR.js

```diff
@@ -70,7 +71,7 @@ ViroMaterials.createMaterials({
     lightingModel: 'Blinn',
   },
 });
-
+// CR * anim label?
```

Comment:
Awesome to see some animations registered to be used! How does this get activated? Is this the activation of the animation? Is there a label that needs to be supplied? Very cool. 🤩